Untitled.ipynb

```diff
@@ -7,6 +7,36 @@
     "Initial state of notebook"
```

Comment:
<p>This seems unchanged!</p>

 _Reply via <a href='https://app.reviewnb.com/slomeks/test-reviewnb/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>